os/drivers/lcd/lcd_dev.c

```diff
@@ -423,13 +424,15 @@ int lcddev_register(struct lcd_dev_s *dev)
 {
 	char devname[16] = { 0, };
 	int ret;
+	char *parm[2];
+	char parm_buf[9]; /* for storing 32 bit address */
```

Comment:
The param and param_buf are difficult to distinguish and to know what they are. Could you rename them to know what they are in name?

Reply:
> The param and param_buf are difficult to distinguish and to know what they are. Could you rename them to know what they are in name?

fixed